app/api/v1/endpoints/seasonal_forecast_list/router.py

```diff
@@ -0,0 +1,25 @@
+from typing import OrderedDict
+from fastapi import APIRouter, HTTPException
+from app.core.webScrape.webscraper import scrape_webpage, mw_seasonal_forecast_list_url
+
+from .schema import (
+    SeasonalForecastListParameters,
+)
+
+router = APIRouter()
+
+
+@router.post("/")
+def get_seasonal_forecast_list(
+    params: SeasonalForecastListParameters,
+) -> OrderedDict:
+        
+    if params.country == "mw":
+        districts = scrape_webpage(mw_seasonal_forecast_list_url)
+        return {"districts": districts}   
+    else:
+        raise HTTPException(status_code=404, detail=f"Seasonal Forecasts not set up '{params.country}'")
```

Comment:
Thanks for adding the exception this, nice to be notified.
Would suggest next call discussing what might be the best strategy for zm